srk/src/expPolynomial.ml

```diff
@@ -2,46 +2,85 @@ open BatPervasives
 open Syntax
 
 module QQX = Polynomial.QQX
-module E = Ring.RingMap(QQ)(QQX)
 module QQMatrix = Linear.QQMatrix
 
-type t = E.t
+module MakeEP 
+  (B : sig 
```

Comment:
Yes. In rational.ml C is a const ring of coefficients. I.e. multivariate polynomials over a number field and variables are symbolic constants. Whereas B are elements of a number field. There they need to be separate, because I can compute algebraic relations among exponentials over a number field, but not w.r.t exponentials where the bases are multivariate polynomials.